payload-schemas/schemas/common/installation.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "$installation",
+  "type": ["object", "null"],
+  "properties": {
+    "id": {
+      "type": "integer"
+    },
+    "node_id": {
+      "type": "string"
+    }
+  },
+  "additionalProperties": false
+}
```

Comment:
Make sure to add the schema files to the files passed by the linter.
There needs to be a new-line here